pkg/dogstatsd/parse_events.go

```diff
@@ -85,7 +84,9 @@ func parseHeader(rawHeader []byte) (eventHeader, error) {
 
 	// Convert title length to workable type and do a basic validity check on value
 	titleLength, err := parseInt64(rawTitleLength)
-	if err != nil || titleLength < 0 {
+	// Before Go 1.17, we can use the following trick to define MaxInt
+	const MaxInt = ^uint(0) >> 1
+	if err != nil || titleLength < 0 || titleLength > int64(MaxInt) {
```

Comment:
As `titleLength` is an `int64`, [there is no possible value that can be greater than `MaxInt`](https://play.golang.org/p/I4wtFzyNunN).
So, `titleLength > int64(MaxInt)` is an antilogy, isn’t it ?